src/autoupdater.ts

```diff
@@ -55,6 +60,34 @@ export class AutoUpdater {
     return isUpdated;
   }
 
+  async handleSchedule(): Promise<number> {
+    const ref = this.config.getValue('GITHUB_REF');
+    const ownerAndRepo = this.config.getValue('GITHUB_REPOSITORY');
```

Comment:
What's the reason for not making these their own functions on the config loader?

Also - if you provided `true` as the second argument to `getValue`, the config loader will automatically throw an error for you (and you can remove the two checks below)

Reply:
No particular reason - I'll add them as their own config functions